ui/apps/platform/src/Containers/NetworkGraph/FlowsSelect.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import { Select, SelectOption } from '@patternfly/react-core';
+
+export type FlowsState = 'active' | 'extraneous';
```

Comment:
So I was a little confused since we use the term `flows` for the extraneous flows and `traffic` for the active traffic. So I think `flows` might not be the best wording for this. I talked to Alan and I think he was good with the term `EdgeState`. Could we change the `Flows` prefix to that?

Reply:
He says `EdgeType` would also work